pgx-examples/spi_srf/src/lib.rs

```diff
@@ -0,0 +1,145 @@
+/*
+Portions Copyright 2019-2021 ZomboDB, LLC.
+Portions Copyright 2021-2022 Technology Concepts & Design, Inc. <support@tcdi.com>
+
+All rights reserved.
+
+Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+*/
+use pgx::TableIterator;
+use pgx::*;
+
+pg_module_magic!();
```

Comment:
Note the above change also requires:
```suggestion
pgx::pg_module_magic!();
```

Reply:
Are you referring to trying to use the namespace? Like using pgx:: ?